orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -819,6 +819,17 @@ public void removeDeadSaltMinions(GatewayConfig gateway) throws CloudbreakOrches
         }
     }
 
+    @Override
+    public Map<String, String> replacePatternInFileOnAllHosts(GatewayConfig gatewayConfig, String file, String pattern, String replace)
```

Comment:
where is this called from?

Reply:
It remained there from the previous implementation (file update instead of full salt update), I think we can leave it there